buildpack.md

```diff
@@ -1,8 +1,8 @@
 # Buildpack Interface Specification
 
-This document specifies the interface between a single lifecycle and one or more buildpacks.
+This document specifies the interface between a program called `lifecycle` and one or more buildpacks.
 
-A lifecycle is a program that uses buildpacks to transform application source code into an OCI image containing the compiled application.
+A `lifecycle` program uses buildpacks to build application source code and pack the compiled application into an OCI image.
```

Comment:
Even if most buildpacks deal with application source code, not all buildplacks do. @nebhale has some Java buildpacks that take compiled JARs for instance.

Reply:
Confirmed, that in fact, other than the [Paketo](https://paketo.io/) build-system buildpacks (only three of dozen or so), all the rest of the buildpacks work against pre-compiled artifacts.  This allows Java users, who typically already have CI systems that build artifacts to repositories, to use those pre-compiled artifacts in their images if they choose.

Reply:
Can I say that buildpack is a build system in a container, and `lifecycle` program is used to mount application directory and get build artifacts out of it?